models/gif_cond_exp_multisynapse.h

```diff
@@ -0,0 +1,467 @@
+
+#include "config.h"
+
+#ifdef HAVE_GSL
+
+// Includes from nestkernel:
+#include "event.h"
+#include "archiving_node.h"
+#include "ring_buffer.h"
+#include "connection.h"
+#include "universal_data_logger.h"
+
+// Includes from gnu gsl:
+#include <gsl/gsl_errno.h>
+#include <gsl/gsl_matrix.h>
+#include <gsl/gsl_odeiv.h>
+
+#include "nest.h"
+
+/* BeginDocumentation
+  Name: gif_cond_exp_multisynapse - Conductance based generalized
+  integrate-and-fire neuron model according to Mensi et al. (2012)
+  and Pozzorini et al. (2015)
+
+  Description:
+
+  gif_cond_exp_multisynapse is the generalized integrate-and-fire neuron
+  according to Mensi et al. (2012) and Pozzorini et al. (2015), with
+  post-synaptic conductances in the form of truncated exponentials.
+
+  This model features both an adaptation current and a dynamic threshold for
+  spike-frequency adaptation. The membrane potential (V) is described by the
+  differential equation:
+
+  C*dV(t)/dt = -g_L*(V(t)-E_L) - eta_1(t) - eta_2(t) - ... - eta_n(t) + I(t)
+
+  where each eta_i is a spike-triggered current (stc), and the neuron model can
+  have arbitrary number of them.
+  Dynamic of each eta_i is described by:
+
+  tau_eta_i*d{eta_i}/dt = -eta_i
+
+  and in case of spike emission, its value increased by a constant (which can be
+  positive or negative):
+
+  eta_i = eta_i + q_eta_i  (in case of spike emission).
+
+  Neuron produces spikes STOCHASTICALLY according to a point process with the
+  firing intensity:
+
+  lambda(t) = lambda_0 * exp[(V(t)-V_T(t)/Delta_V)]
+
+  where V_T(t) is a time-dependent firing threshold:
+
+  V_T(t) = V_T_star + gamma_1(t) + gamma_2(t) + ... + gamma_m(t)
+
+  where gamma_i is a kernel of spike-frequency adaptation (sfa), and the neuron
+  model can have arbitrary number of them.
+  Dynamic of each gamma_i is described by:
+
+  tau_gamma_i*d{gamma_i}/dt = -gamma_i
+
+  and in case of spike emission, its value increased by a constant (which can be
+  positive or negative):
+
+  gamma_i = gamma_i + q_gamma_i  (in case of spike emission).
+
+  Note that in the current implementation of the model (as described in [1] and
+  [2]) the values of eta_i and gamma_i are affected immediately after spike
+  emission. However, GIF toolbox (http://wiki.epfl.ch/giftoolbox) which fits
+  the model using experimental data, requires a different set of eta_i and
+  gamma_i. It applies the jump of eta_i and gamma_i after the refractory period.
+  One can easily convert between q_eta/gamma of these two approaches:
+  q_eta_giftoolbox = q_eta_NEST * (1 - exp( -tau_ref / tau_eta ))
+  The same formula applies for q_gamma.
+
+  On the postsynapic side, there can be arbitrarily many synaptic time constants
+  (gif_psc_exp has exactly two: tau_syn_ex and tau_syn_in). This can be reached
+  by specifying separate receptor ports, each for a different time constant. The
+  port number has to match the respective "receptor_type" in the connectors.
```

Comment:
I think it would be a good thing to develop this part a bit, perhaps giving a quick example like:
```python
''' Create neuron, voltmeter and poisson_generator '''
n = nest.Create("gif_cond_exp_multisynapse", params={"taus_syn": [1.3,  4.5, 0.8]}) # 3 synapses/ports
pg = nest.Create("poisson_generator", 3, params={"rate": 10000.}) # poisson generator for each synapse
vm = nest.Create("voltmeter")
''' Connect voltmeter ''''
nest.Connect(vm, n, syn_spec={"receptor_type": 0})
''' Connect poisson generator '''
nest.Connect([pg[0]], n, syn_spec={"receptor_type": 1})
nest.Connect([pg[1]], n, syn_spec={"receptor_type": 2})
nest.Connect([pg[2]], n, syn_spec={"receptor_type": 3})

This would make the description self-sufficient for model use...